tests/test_sentinel_privileges.py

```
@@ -10,7 +10,15 @@ def test_sentinel_privileges_enforcement(tmp_path, monkeypatch):
     test_config_file = test_config_dir / "config.json"
 
     # Mock CONFIG_DIR and CONFIG_FILE in deepcli.deepcli.core
-    import deepcli.deepcli.core as dc
+    def _get_dc():
+        try:
+            import deepcli.deepcli.core as dc
```

Pylint (4.0.7)

[convention] Import outside toplevel (deepcli.deepcli.core)

(C0415)